ts-embed_assets/README.md

```diff
@@ -0,0 +1,61 @@
+# Embedding Assets Sample App
+
+## Run the app locally
+```sh
+ts-node index.ts
+```
+
+Test APIs with:
+```
+$ curl localhost:3000/hello
+hi%
+
+$ curl localhost:3000/static/hello.txt
+Hello Klotho!
+```
+
+## Compile and Deploy with Klotho
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho . --app $USER-ts-embed-assets -p aws
+
+# Go into the compiled directory
+cd compiled
+
+# If you didn't set the aws region as indicated in the compiler output, do that now
+pulumi config set aws:region YOUR_REGION -s $USER-ts-embed-assets
```

Comment:
Do we want to add instructions for the aws creds, or are we assuming that they're in the appropriate env vars?

In particular, I'd argue for at least an optional instruction for help with profiles:

```
# Optional: If you've set up an AWS CLI profile (via `aws configure --profile`), you can tell Pulumi to use it:
pulumi config set aws:profile $yourAwsProfileName
```